app/server.go

```diff
@@ -48,6 +48,7 @@ var MaxNotificationsPerChannelDefault int64 = 1000000
 type Server struct {
 	Store           store.Store
 	WebSocketRouter *WebSocketRouter
+	AppInitialized  sync.Once
```

Comment:
Does this need to be exported ?

Idiomatic Go also suggests that it should be named - "appInitializedOnce".

Reply:
Actually is part of the server, if we want to move it out of the app layer, the expectation is this is public, not private.